datastreamer/streamserver.go

```diff
@@ -37,19 +37,27 @@ type CommandError uint32
 // EntryTypeNotFound is the entry type value for CmdEntry/CmdBookmark when entry/bookmark not found
 const EntryTypeNotFound = math.MaxUint32
 
+// Entry type constants for data stream entries (internal definitions)
+const (
+	internalEntryTypeL2Block EntryType = 2 // L2Block entry type
```

Comment:
why don't use `EntryTypeL2Block' ?

Reply:
Because this is a type used by the DS service, and DS itself doesn't know what type it is, a prefix was added.